components/components/navigation/menu-main/menu-main.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+* @file
+* This file creates the HTML for the main menu.
+*/
+#}
+
+<nav role="navigation" aria-labelledby="block-umdrupal-main-menu-menu" id="block-umdrupal-main-menu" class="block block-menu navigation menu-main__wrapper">
+
+  <h2 class="menu-main__title visually-hidden" id="block-umdrupal-main-menu-menu">Main navigation</h2>
+
+  <button type="button" name="menu_toggle" class="menu-main__toggle" aria-label="Toggle the menu">{% include 'menu-icon.svg' %}</button>
+
+  <ul class="menu-main">
+    <li class="menu-main__item menu-main__item--active-trail">
+      <a href="/" data-drupal-link-system-path="<front>" class="menu-main__link is-active">Home</a>
```

Comment:
Could we pass the data from a JSON file instead of hardcoding them here?

Reply:
Yes, I will do.